helm/Makefile

```diff
@@ -1,4 +1,5 @@
 export VERSION:=$(shell ../version.sh)
```

Comment:
can we remove `VERSION`? and completely replace with `VERSION_TAG`?

Reply:
`VERSION` is still used for Helm packaging, and it has to be semver-compatible, so it can't begin with a `v` like the tags themselves.